lib/eu/hydrologis/smash/forms/form_smash_utils.dart

```diff
@@ -11,6 +11,7 @@ import 'package:smash/eu/hydrologis/smash/project/images.dart';
 import 'package:smash/eu/hydrologis/smash/project/objects/images.dart';
 import 'package:smash/eu/hydrologis/smash/project/objects/notes.dart';
 import 'package:smashlibs/smashlibs.dart';
+import '../../../../generated/l10n.dart';
```

Comment:
Hi, I am trying to keep imports package based. Would be great to do that also for the l10n.

Reply:
This wasn't really clear to me and I also thought a package would be better. I just followed your first example though, because I thought this would be intentional.